### PR TITLE
README.files correction: QUICKSTART.rst has moved

### PR DESCRIPTION
**Pull request by awallace-cray**

### CODE REVIEW

**Comment by awallace-cray** [comment]
@bradcray - this is a proposal, feel free to decline or suggest alternatives. Its optional. Thanks.

**Comment by bradcray** [comment]
This sounds like a @ben-albrecht item since he moved it.

**Comment by ben-albrecht** [comment]
We should probably include all the sub-directories of `doc/` with descriptions or none of them. Is this README intended for users of the release tarball, i.e. should we assume the docs are built?

File: README.files [hunk 1]
```diff
@@ -16,12 +16,13 @@ further information about the files and directories that they contain.
...
   LICENSE.chapel       : license statement for the Chapel implementation itself
   PERFORMANCE.md       : notes about Chapel performance
-  QUICKSTART.rst       : quick-start instructions
   README.files         : this file
   Makefile             : top-level Makefile for building things
   bin/                 : location of the compiler (chpl) once it's been built
   compiler/            : source code for the compiler
-  doc/                 : documentation and READMEs
+  doc/                 : documentation and READMEs; especially,
+    language/            : introduction to the Chapel language
```
Comment (ben-albrecht) [inline README.files]: Can we vertically align `:`'s?

**Comment by bradcray** [comment]
> We should probably include all the sub-directories of doc/

@ben-albrecht, this reminds me that I still feel like the doc/ directory as it's presented in the release has backslid from the last release in terms of its accessibility (something that I still hope we can address for this release -- recall my emails about no top-level entry point to get started with, lack of directory symmetry between the html and rst versions of the docs).  For this reason, I wouldn't spend too much time documenting the doc/ directory's structure here, now.

**Comment by awallace-cray** [comment]
What Brad said - just remove QUICKSTART.rst, only

**Pull request merged**